packages/table/preview/mutableTable.tsx

```diff
@@ -603,6 +603,7 @@ export class MutableTable extends React.Component<{}, IMutableTableState> {
                 {this.renderSwitch("Loading state", "showColumnHeadersLoading")}
                 {this.renderSwitch("Menus", "showColumnMenus")}
                 <h6>Interactions</h6>
+                {this.renderSwitch("Custom Headers", "enableColumnCustomHeaders")}
```

Comment:
I'd say this belongs in the `Display` section. 🤓

Reply:
Well if we do that, we should disable "Editing" with a note about this.

Reply:
@gscshoyru you can make the editing toggle dependent on this one:

```tsx
{this.renderSwitch("Editing", "enableColumnNameEditing", "enableColumnCustomHeaders", false)}
``` 

Then the editing switch will be disabled and show a tooltip that says:
`Requires enableColumnCustomHeaders=false` on hover.

Reply:
Oh, and also, `Sentence cased` please.